### PR TITLE
Include Fixed Network & Native Asset Icons

### PR DESCRIPTION
**Pull request by rubenmarcus**
this p/r intends to address the correct icons for the chains used on the lib

### CODE REVIEW

File: src/icons.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+export {ETHER_ICON,ETH_SMALL,BASE_SMALL,ARBITRUM_SMALL,OPT_SMALL,POL_SMALL, POL_ICON};
```
Comment (bh2smith) [inline src/icons.ts:20]:
```suggestion
export {ETHER_ICON, ETH_SMALL, BASE_SMALL, ARBITRUM_SMALL, OPT_SMALL, POL_SMALL, POL_ICON};

```

File: src/constants.ts [hunk 2]
```diff
@@ -15,7 +14,7 @@ export const CHAIN_INFO: Record<number, ChainInfo> = {
...
+    icon:OPT_SMALL,
```
Comment (bh2smith) [inline src/constants.ts:17]:
```suggestion
    icon: OPT_SMALL,
```

File: src/constants.ts [hunk 3]
```diff
@@ -37,19 +36,20 @@ export const CHAIN_INFO: Record<number, ChainInfo> = {
...
+    icon:BASE_SMALL,
```
Comment (bh2smith) [inline src/constants.ts:46]:
```suggestion
    icon: BASE_SMALL,
```

**Review by bh2smith — COMMENTED** [review]
This looks pretty good. Now that we have three files on the top layer of the source directory that are exclusively related to network data, I would propose that we create a network directory and move these three files into it as:
```
network/
    index.ts (this is the current network file)
    data.ts (consolidate all of the content from icons.ts and constants.ts only exporting CHAIN_INFO)
```

I don't think there need to be two files dedicated to icons (i.e. constants and icons) and the icons, don't seen to require being exported if they are all in the same file). and the only icon that is used more than once is `ETHER_ICON` (so the old pattern would have been fine too). 

If we want to be more clear about which network belongs to which chainId, maybe we can add comment strings above each chainId. 

There are several formatting issues in this code that could be resolved with `yarn fmt`.


:warning: Don't forget to `export * from "./networks";` in `src/index.ts` if it isn't already.

**Comment by rubenmarcus** [comment]
> This looks pretty good. Now that we have three files on the top layer of the source directory that are exclusively related to network data, I would propose that we create a network directory and move these three files into it as:
> 
> ```
> network/
>     index.ts (this is the current network file)
>     data.ts (consolidate all of the content from icons.ts and constants.ts only exporting CHAIN_INFO)
> ```
> 
> I don't think there need to be two files dedicated to icons (i.e. constants and icons) and the icons, don't seen to require being exported if they are all in the same file). and the only icon that is used more than once is `ETHER_ICON` (so the old pattern would have been fine too).
> 
> If we want to be more clear about which network belongs to which chainId, maybe we can add comment strings above each chainId.
> 
> ⚠️ Don't forget to `export * from "./networks";` in `src/index.ts` if it isn't already.
> 
> There are some formatting issues highlighted inline that could all be resolved with `yarn fmt`. This was only passing because a missing check in our CI (resolved by #134).
> 
> Would be nice to see a PR description and a more thoughtful title (correctly capitalized).

changes done

**Review by bh2smith — APPROVED** [review]
Looks great.

**Pull request merged**